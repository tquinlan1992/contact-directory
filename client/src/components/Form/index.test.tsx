import Enzyme, { render } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import { Form, TextInput, FormButton } from ".";
Enzyme.configure({ adapter: new Adapter() });

describe("Form components", () => {
  it("renders all the form fields", () => {
    const wrapper = render(
      <Form>
        <TextInput
          onChange={jest.fn()}
          label="text_input_label"
          value="text_input_value"
        />
        <FormButton onClick={jest.fn()} />
      </Form>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
