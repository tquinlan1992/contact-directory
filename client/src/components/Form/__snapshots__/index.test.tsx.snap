// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form components renders all the form fields 1`] = `
<div
  class="form"
>
  <div
    class="form-input-container"
  >
    <label
      class="input-label"
    >
      text_input_label
    </label>
    <input
      type="text"
      value="text_input_value"
    />
  </div>
  <button />
</div>
`;
